numpy/core/include/numpy/ndarrayobject.h

```diff
@@ -170,14 +170,19 @@ extern "C" CONFUSE_EMACS
                                             (k)*PyArray_STRIDES(obj)[2] + \
                                             (l)*PyArray_STRIDES(obj)[3]))
 
+/* Move to arrayobject.c once PyArray_XDECREF_ERR is removed */
 static NPY_INLINE void
 PyArray_DiscardWritebackIfCopy(PyArrayObject *arr)
 {
     if (arr != NULL) {
+        PyArrayObject_fields *fa = (PyArrayObject_fields *)arr;
         if ((PyArray_FLAGS(arr) & NPY_ARRAY_WRITEBACKIFCOPY) ||
             (PyArray_FLAGS(arr) & NPY_ARRAY_UPDATEIFCOPY)) {
```

Comment:
Needs more indentation to separate it from the block. Alignment isn't desirable here.